docs/ism/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Index State Management
```

Comment:
Why do we have ISM here?

Reply:
This is to change the order of the documents for PPL. It's in the ISM section.